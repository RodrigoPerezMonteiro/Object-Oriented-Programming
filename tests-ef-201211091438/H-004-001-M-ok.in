7
3
Feel light
4
linguado grelhado
4
salada mista
4
alface
1
0
3
All you want
4
linguado grelhado
4
salada mista
4
alface
1
0
3
All Pizza
4
alface
1
0
0
4
testH4.dat
0
