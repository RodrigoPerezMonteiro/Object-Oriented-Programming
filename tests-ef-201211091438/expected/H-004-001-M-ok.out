Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: VEG|alface|5|80|AVAILABLE
NOVEG|linguado grelhado|10|20|AVAILABLE
VEG|salada mista|2|0|AVAILABLE
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: VEG|alface|5|0|AVAILABLE
NOVEG|linguado grelhado|6|0|AVAILABLE
VEG|salada mista|3|0|AVAILABLE
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Identificador do restaurante (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Identificador do prato do dia (nome): Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: VEG|alface|3|0|AVAILABLE
Gestão de um Restaurante
1 - Visualizar pratos do dia
2 - Criar prato do dia
3 - Calcular calorias
4 - Disponibilizar prato do dia
5 - Indisponibilizar prato do dia
6 - Alterar desconto
0 - Sair
Escolha uma opção: Gestão de Restaurantes
1 - Visualizar restaurantes
2 - Registar restaurante
3 - Menu de um restaurante
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Guardar ficheiro como: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 